text/0000-location.md

```diff
@@ -0,0 +1,417 @@
+(Organizations and agents are defined by the Pike smart contract.)
+
+If the Grid setting grid.location.allow_delete is set to false, this transaction
+is invalid. The default value for grid.location.allow_delete is true. This 
+setting is stored using the Sawtooth Settings smart contract, more information
+can be found here.
+
+Validation requirements:
+- If a location with location_id does not exist the transaction is invalid.
+- The signer of the transaction must be an agent in the Pike state and must 
+belong to an organization in Pike state, otherwise the transaction is invalid.
+- The owner in the location must match the organization that the agent belongs 
+to, otherwise the transaction is invalid.
+- The agent must have the permission “can_delete_location” for the organization
+otherwise the transaction is invalid.
+
+The inputs for LocationDeleteAction must include:
+- Grid address of the Agent submitting the transaction
+- Grid address of the Organization the Location is being deleted for Grid 
+address of the Location to be deleted
+
+The outputs for LocationDeleteAction must include:
+- Grid address of the Location to be deleted
+
+### Defined GS1 Properties
+
+This Location RFC defines required and optional attributes for GS1 Locations 
+(see table below). This implementation does not include attribution for specific
+industries, financial/tax account information, a physical location extension, or
+digital locations and should be implemented with supplemental RFC(s).
+
+**_REQUIRED FIELDS_**
+|GS1 Common Name|GS1 Attribute Name|Description|Example|Data Type|Min|Max|
+|---------------|------------------|-----------|-------|----|---|---|
+|Location Name 1|locationName|The name of the facility being described.|"Sunny Fresh Foods"|STRING|1|80|
+|Description|locationDescription|Free text, 178 characters.|"A Cargill production facility dedicated to serving high-quality egg products across various markets."|STRING| | |
+|Location Type|locationType|Multiple types allowed. All Suppliers: Org Entity, Order From, Remit To, Ship To. Healthcare Providers: Bill To, Deliver To, Order By, Order From, Org Entity, Paid By, Recall, Remit To, Ship From, Ship To.|"Ship From"|ENUM|7|48|
+|Address Line 1|addressLine1|The primary street address for your location. The USPS address is validated if Country = United States.| "206 W 4th Street"|STRING|1|80|
+|City|city|Name of the city of your location. The USPS address is validated if Country = United States.|"Monticello"|STRING|1|35|
+|State or Region|stateOrRegion|The state, province, or region using the standard two-letter abbreviation specified in ISO 3166-2:1998 country subdivision code [16].|"MN"|STRING|1|3|
+|Postal Code|postalCode|The ZIP or other postal code.|"55362-8524"|STRING|1|10|
+|Country|country|Country of your location. Spell out, do not use abbreviations.|"United States"|STRING|2|80|
+|Latitude & Longitude|LAT_Long| |"44.986656, -93.258133"|LAT_LONG| | |
```

Comment:
is `LAT_Long` a valid GS1 attribute? I was able to find documentation for latitude and longitude separately but I am unsure if there is an attribute combining those values.

Reply:
Good call out. The GS1 documentation I found defined latitude and longitude separately. @vaporos suggested the idea that we merge them based on our current LAT_LONG property definition (see above) and I updated the PR to reflect this change. Talking out loud here - is it best practice to align the GS1 Attributes defined within the RFC with the GS1 data standard/specification and then adjust the Data Type/property definition to support the need?

Reply:
I feel like these convey the same idea and use native Grid typing for broader platform alignment.  I prefer the LAT_LONG format.

Reply:
I agree with @davececchi. It's not necessary to have a 1 for 1 equivalent, as it would be mapped when sharing data in GS1 format anyway.

Reply:
Should the GS1 attribute name here still be "latitude" and "longitude" to match the spec (I'm assuming that this was correct previously, I'm not looking at the spec ATM)? For example:

|Latitude, Longitude|latitude, longitude| |"44.986656, -93.258133"|LAT_LONG| | |

Reply:
yes - should match the spec name while adopting Grid typing.

Reply:
GS1 US Standards team member here...

Currently, they are captured separately but that may not always be the case as there is an effort to review how location attribute data is in use. The [GS1 web vocabulary](https://www.gs1.org/voc/GeoCoordinates) has some guidance on how Latitude & Longitude should be captured currently for location attribute data but reminder that this may not be case long-term.

As @auntmelmel notes above, it may not be necessary to mirror the syntax as along as the coordinates can be mapped to the GS1 syntax when being shared.

Reply:
I updated the RFC to reflect the spec names and Grid Data Type; believe this thread may be resolved.
|Latitude, Longitude|latitude, longitude| |"44.986656, -93.258133"|LAT_LONG| | |

Reply:
What was the rationale for changing the name in Product to ProductType I wonder? Presumably it was done at implementation time if it is still called ProductNamespace in the RFC.